### PR TITLE
Replace Madoko language spec and PSA spec with Asciidoc

### PR DESCRIPTION
**Pull request by jafingerhut**

### CODE REVIEW

**Comment by jafingerhut** [comment]
OK, I promise not to approve any other PRs until after this one is merged in :-)

**Review by rcgoodfellow — APPROVED** [review]

**Comment by jafingerhut** [comment]
@rcgoodfellow Any other reviews you want on this one, or good to merge now?

**Comment by rcgoodfellow** [comment]
Let's get this landed! We can deal with any issues that come up with follow-on PRs. But I think it's important to get the main branch on the Asciidoc train.

**Pull request merged**
